docs/source/image_classification.mdx

```diff
@@ -75,6 +75,9 @@ You can verify the transformation worked by indexing into the `pixel_values` of
 >>> plt.imshow(img)
 ```
 
+To learn how to leverage 🤗 datasets end-to-end for training an image classification model, refer to
+[this notebook](https://github.com/huggingface/notebooks/blob/main/examples/image_classification.ipynb).
```

Comment:
Would it be possible to link this straight to a Colab notebook so users can start running the code if they'd like (same comment for the object detection doc)?